concrete/single_pages/dashboard/system/files/thumbnails.php

```diff
@@ -126,8 +137,9 @@
     <tr>
         <td><a href="<?=$view->action('edit', $type->getID())?>"><?=$type->getHandle()?></a></td>
         <td><?=$type->getDisplayName()?></td>
-        <td><?=$type->getWidth()?></td>
+        <td><?=($type->getWidth()) ? $type->getWidth() : '<span class="text-muted">' . t('Automatic') . '</span>' ?></td>
```

Comment:
@mlocati again agree but I just copied the line that was already there for height. Usually, when the core team does something a certain way I leave it like that in case it's the way they like it. DO you think I should remove the parenthesis?